src/App.jsx

```diff
@@ -0,0 +1,29 @@
+import "./styles.css";
+import { VideoListing } from "./pages/Videos/VideoListing";
+import { Playlists } from "./pages/Playlists/Playlists";
+import { PlaylistDetail } from "./pages/Playlists/PlaylistDetail";
+import { Routes, Route } from "react-router-dom";
+import { NavigationBar } from "./components/NavigationBar";
+import { VideoDetail } from "./pages/Videos/VideoDetail";
+import { PageNotFound } from "./pages/PageNotFound";
+import { History } from "./pages/History/History";
+import { SearchResults } from "./pages/SearchResults";
+import { ToastContainer } from "react-toastify";
```

Comment:
Instead of doing separate imports for the same folder, you can use `index.js` inside page folder and re-export all nested components inside pages folder
e.g. 
```suggestion
import { VideoListing, Playlists, PlaylistDetail,... } from "./pages";
```